lib/danger/ci_source/support/pull_request_finder.rb

```diff
@@ -151,8 +151,15 @@ def client
 
       when :github
         require "octokit"
-        Octokit::Client.new(access_token: ENV["DANGER_GITHUB_API_TOKEN"], api_endpoint: api_url)
-
+        access_token = ENV["DANGER_GITHUB_API_TOKEN"]
+        bearer_token = ENV["DANGER_GITHUB_BEARER_TOKEN"]
+        if bearer_token && !bearer_token.empty?
+          Octokit::Client.new(bearer_token: bearer_token, api_endpoint: api_url)
+        elsif access_token && !access_token.empty?
+          Octokit::Client.new(access_token: access_token, api_endpoint: api_url)
+        else
+          raise "No API token given, please provide one using `DANGER_GITHUB_API_TOKEN` or `DANGER_GITHUB_BEARER_TOKEN`"
```

Comment:
before this patch, this implementation doesn't care existence of `access_token`.
It's better to be more strict for user-friendly.